CHANGELOG.md

```diff
@@ -45,23 +45,23 @@
 * [#1566](https://github.com/stripe/stripe-java/pull/1566) Update generated code
   * Add support for `zip_payments` on `Account.capabilities`, `AccountCreateParams.capabilities`, and `AccountUpdateParams.capabilities`
   * Add support for `zip` on `Charge.payment_method_details`, `PaymentIntent.payment_method_options`, `PaymentIntentConfirmParams.payment_method_data`, `PaymentIntentConfirmParams.payment_method_options`, `PaymentIntentCreateParams.payment_method_data`, `PaymentIntentCreateParams.payment_method_options`, `PaymentIntentUpdateParams.payment_method_data`, `PaymentIntentUpdateParams.payment_method_options`, `PaymentMethodCreateParams`, `PaymentMethodUpdateParams`, `PaymentMethod`, `SetupIntentConfirmParams.payment_method_data`, `SetupIntentCreateParams.payment_method_data`, and `SetupIntentUpdateParams.payment_method_data`
-  * Add support for new value `zip` on enums `CheckoutSessionCreateParams.payment_method_types[]` and `PaymentMethodCreateParams.type`
+  * Add support for new value `zip` on enums `checkout.SessionCreateParams.payment_method_types[]` and `PaymentMethodCreateParams.type`
   * Add support for new value `zip` on enums `CustomerListPaymentMethodsParams.type` and `PaymentMethodListParams.type`
   * Add support for new value `zip` on enums `PaymentIntentConfirmParams.payment_method_data.type`, `PaymentIntentCreateParams.payment_method_data.type`, `PaymentIntentUpdateParams.payment_method_data.type`, `SetupIntentConfirmParams.payment_method_data.type`, `SetupIntentCreateParams.payment_method_data.type`, and `SetupIntentUpdateParams.payment_method_data.type`
 
 ## 22.20.0 - 2023-05-19
 * [#1563](https://github.com/stripe/stripe-java/pull/1563) Update generated code
-  * Add support for `subscription_update_confirm` and `subscription_update` on `BillingPortal.Session.flow` and `BillingPortalSessionCreateParams.flow_data`
-  * Add support for new values `subscription_update_confirm` and `subscription_update` on enum `BillingPortalSessionCreateParams.flow_data.type`
+  * Add support for `subscription_update_confirm` and `subscription_update` on `BillingPortal.Session.flow` and `billing_portal.SessionCreateParams.flow_data`
```

Comment:
If we're trying to make this match the package name, we should rename `billing_portal` -> `billingportal`